packages/d3fc-webgl/README.md

```diff
@@ -2,15 +2,403 @@
+
+Sets the *attributeBufferBuilder* to be used to create the buffer for the `aOpenValue` shader attribute.
+
+<a name="webglSeriesOhlc_highValueAttribute" href="#webglSeriesOhlc_highValueAttribute">#</a> *webglSeriesOhlc*.**highValueAttribute**(*attributeBufferBuilder*)
+
+Sets the *attributeBufferBuilder* to be used to create the buffer for the `aHighValue` shader attribute.
+
+<a name="webglSeriesOhlc_lowValueAttribute" href="#webglSeriesOhlc_lowValueAttribute">#</a> *webglSeriesOhlc*.**lowValueAttribute**(*attributeBufferBuilder*)
+
+Sets the *attributeBufferBuilder* to be used to create the buffer for the `aLowValue` shader attribute.
+
+<a name="webglSeriesOhlc_closeValueAttribute" href="#webglSeriesOhlc_closeValueAttribute">#</a> *webglSeriesOhlc*.**closeValueAttribute**(*attributeBufferBuilder*)
+
+Sets the *attributeBufferBuilder* to be used to create the buffer for the `aCloseValue` shader attribute.
+
+<a name="webglSeriesOhlc_bandwidthAttribute" href="#webglSeriesOhlc_bandwidthAttribute">#</a> *webglSeriesOhlc*.**bandwidthAttribute**(*attributeBufferBuilder*)
+
+Sets the *attributeBufferBuilder* to be used to create the buffer for the `aBandwidth` shader attribute.
+
+<a name="webglSeriesOhlc_definedAttribute" href="#webglSeriesOhlc_definedAttribute">#</a> *webglSeriesOhlc*.**definedAttribute**(*attributeBufferBuilder*)
+
+Sets the *attributeBufferBuilder* to be used to create the buffer for the `aDefined` shader attribute.
+
+Values calculated from the given *attributeBufferBuilder* should either be `1` (to indicate the data point is defined) or `0` (to indicate the data point is not defined).
+
+#### Point
+
+<a name="webglSeriesPoint" href="#webglSeriesPoint">#</a> fc.**webglSeriesPoint**()
+
+Used to construct a new WebGL Point series.
+
+<a name="webglSeriesPoint_type" href="#webglSeriesPoint_type">#</a> *webglSeriesPoint*.**type**(*symbolTypeShader*)
+
+If *symbolTypeShader* is specified, sets the symbol type shader and returns this series. If *symbolTypeShader* is not specified, returns the current symbol type shader.
+
+A *symbolTypeShader* can be generated using the [webglSymbolMapper](Link to webglSymbolMapper).
+
+<a name="webglSeriesPoint_crossValueAttribute" href="#webglSeriesPoint_crossValueAttribute">#</a> *webglSeriesPoint*.**crossValueAttribute**(*attributeBufferBuilder*)
+
+Sets the *attributeBufferBuilder* to be used to create the buffer for the `aCrossValue` shader attribute.
+
+<a name="webglSeriesPoint_mainValueAttribute" href="#webglSeriesPoint_mainValueAttribute">#</a> *webglSeriesPoint*.**mainValueAttribute**(*attributeBufferBuilder*)
+
+Sets the *attributeBufferBuilder* to be used to create the buffer for the `aMainValue` shader attribute.
+
+<a name="webglSeriesPoint_sizeAttribute" href="#webglSeriesPoint_sizeAttribute">#</a> *webglSeriesPoint*.**sizeAttribute**(*attributeBufferBuilder*)
+
+Sets the *attributeBufferBuilder* to be used to create the buffer for the `aSize` shader attribute.
```

Comment:
What units?

Reply:
I think this is a general **point** (🥁), anytime we have a value that isn't scaled by x/y scale we should give a unit